### ISSUE TITLE
negative sampling triples

### ISSUE THREAD

**Issue by naivblue01** [hsientzucheng/MKR.PyTorch#2]
hi hsientzucheng

i review MRK paper and github code recently for recommendation system
but l can't find negative sampling triples and "scores(positive triple)  - scores(negative triple)" loss function
I would be very grateful if you could answer me
thankyou
